### DISCUSSION TITLE
Bug in Vercel AI SDK 3.1.1

### DISCUSSION THREAD

**Discussion by BishalSaha** [vercel/community#6747] [Bug Report]
### Page to Investigate

https://sdk.vercel.ai/examples/next-app/tools/render-interface-during-function-call

### Steps to Reproduce

The code for example is wrong or it has bugs, getting error -

Unhandled Runtime Error
Error: Client Functions cannot be passed directly to Server Functions. Only Functions passed from the Server can be passed back again.

Source
src/app/page.tsx (31:61) @ onClick

  29 |         <button
  30 |           onClick={async () => {
  31 |             const { messages } = await continueConversation([
                                                                                                                    ^
  32 |               ...conversation,
  33 |               { role: "user", content: input },
  34 |             ]);

**Comment by ndukachukz** [comment]
Hello please where you able to fix this? i am having the same error

[a comment by an automated account is not shown]
